plugins/globalias/globalias.plugin.zsh

```diff
@@ -4,8 +4,11 @@ globalias() {
    zle self-insert
 }
 zle -N globalias
-bindkey -e " " globalias
-bindkey -v " " globalias
-bindkey -e "^ " magic-space           # control-space to bypass completion
-bindkey -v "^ " magic-space
-bindkey -M isearch " " magic-space # normal space during searches
```

Comment:
I do get that globalias is run instead of adding a space when I'm on `isearch` mode. Perhaps that's why I'm on emacs mode by default, and that messes it up. If I readd this back it works like I'd expect.